common/src/main/java/org/wso2/testgrid/common/ShellExecutor.java

```diff
@@ -131,4 +132,53 @@ public int executeCommand(String command) throws CommandExecutionException {
             executor.shutdownNow();
         }
     }
+
+    /**
+     * Executes a shell command.
+     *
+     * @param command Command to execute
+     * @return boolean for successful/unsuccessful command execution (success == true)
+     * @throws CommandExecutionException if an {@link IOException} occurs while executing the command
+     */
+    @SuppressFBWarnings("RV_RETURN_VALUE_IGNORED_BAD_PRACTICE")
```

Comment:
Why we suppress the bad practice here? Better explain the reason in the annotation as follows:
`@SuppressFBWarnings(value = "RV_RETURN_VALUE_IGNORED_BAD_PRACTICE",
            justification = "This is my justification")`

Reply:
Correct, the suppression was done since the `ExecutorService#submit` requires assigning to a Future object. This is required only if we reuse the result of `ExecutorService.submit`. Since this can be considered a fire-and-forget, we can use the method Executor#excute. Made changes in e4af8be